src/lib.rs

```diff
@@ -15,6 +15,17 @@ extern crate regex;
 use proc_macro::{TokenStream, TokenTree::*};
 use regex::Regex;
 
+/// Adds ANSI escape codes to a formatting string, allowing ANSI colors to be set at compile time instead of runtime.
+/// 
+/// # Example
+/// 
+/// ```rust
+/// use ansiform::ansi;
+/// 
```

Comment:
Yes, but use would usually do it with a `use` statement in Rust 2018 . That is why I left it in the documentation.
I can remove it – let me know what you think.

Reply:
Yes, that's the usage I usually use, as nearly everything I've done has been in Rust 2018. Because of the simplicity of this crate, though, I was hoping to make it compatible with Rust 2015.